umm-spec-lib/src/cmr/umm_spec/xml_to_umm_mappings/dif10.clj

```diff
@@ -30,11 +30,11 @@
 
 (defn- parse-characteristics
   [el]
-  (seq (remove nil? 
-    (map char-data-type-normalization/normalize-data-type
-      (remove nil?
-        (for [characteristic (select el "Characteristics")]
-          (fields-from characteristic :Name :Description :DataType :Unit :Value)))))))
+  (seq (remove nil?
+         (map char-data-type-normalization/normalize-data-type
```

Comment:
Indentation: the left parens of the `map` should be directly under the `r` of `remove` in the line above ...